src/Apps/Order/Routes/Review/index.tsx

```diff
@@ -129,24 +129,28 @@ export const ReviewRoute: FC<ReviewProps> = props => {
           o => o.lineItems?.edges?.[0]?.node?.artwork?.slug
         )
 
-        if (
-          order.mode === "OFFER" &&
-          isEigen &&
-          order.source === "artwork_page"
-        ) {
-          window?.ReactNativeWebView?.postMessage(
-            JSON.stringify({
-              key: "goToInboxOnMakeOfferSubmission",
-              orderCode: order.code,
-              message: `The seller will respond to your offer by ${order.stateExpiresAt}. Keep in mind making an offer doesn’t guarantee you the work.`,
-            })
-          )
-          // We cannot expect Eigen to respond all the time to messages sent from the webview
-          // a default fallback page is safer for old/broken Eigen versions
-          setTimeout(() => {
-            router.push(`/orders/${order.internalID}/status`)
-          }, 500)
-          return
+        if (isEigen) {
+          if (order.mode === "OFFER" && order.source === "artwork_page") {
+            window?.ReactNativeWebView?.postMessage(
+              JSON.stringify({
+                key: "goToInboxOnMakeOfferSubmission",
+                orderCode: order.code,
+                message: `The seller will respond to your offer by ${order.stateExpiresAt}. Keep in mind making an offer doesn’t guarantee you the work.`,
+              })
+            )
+            // We cannot expect Eigen to respond all the time to messages sent from the webview
+            // a default fallback page is safer for old/broken Eigen versions
+            setTimeout(() => {
+              router.push(`/orders/${order.internalID}/status`)
+            }, 500)
+            return
+          }
+
+          if (order.mode === "BUY") {
```

Comment:
I'm from mobile phone here(ooo for the next 2 weeks) but I would recommend a fallback for older Eigen versions that have no interaction with this new dispatched message. Maybe moving line 143 block down?

Reply:
what will happen if its an older version eigen? 🤔 the message will be posted, but no interaction will happen, right?

if so, that is totally fine for us.

Reply:
Yep, if that's the expected it's fine!